tchannel/serializer/json.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import json
+
+from tchannel.schemes import JSON
+
+
+class JsonSerializer(object):
+    def __init__(self):
+        self.name = JSON
+
+    def deserialize_header(self, obj):
+        if not obj:
+            return None
+        return json.loads(obj)
+
+    def serialize_header(self, obj):
+        if not obj:
+            obj = {}
+        return json.dumps(obj)
+
+    def deserialize_body(self, obj):
+        if not obj:
+            return None
+        return json.loads(obj)
+
+    def serialize_body(self, obj):
+        if not obj:
+            obj = {}
```

Comment:
Same as above. 

Also, we allow arbitrary JSON-like objects. I could pass an empty string there. That's a valid object that can be converted to JSON, but this will convert my `""` to `"{}"`.